pkgs/by-name/m1/m1-terraform-provider-helper/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule {
+  pname = "m1-terraform-provider-helper";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "kreuzwerker";
+    repo = "m1-terraform-provider-helper";
+    rev = "0.9.0";
+    sha256 = "0plng7kz768v1km8mxw7jzsrvpxfzg991lrwwndpkgnyk2m28cnv";
+  };
+
+  # Only build from the root package
+  subPackages = [ "." ];
+
+  vendorHash = "sha256-+UzPs1kiapYy5WMwx6EkQBlBs3DpWhzaQcYqdlzH94U=";
+
+  # Use custom build command from Makefile
+  buildPhase = ''
+    make build-release VERSION=$version
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp dist/release/m1-terraform-provider-helper $out/bin/
+  '';
+
+  # testtarget in makefile relies on git
+  doCheck = false;
```

Comment:
Does using `nativeCheckInputs = [ git ];` allow the checkPhase to pass?

Reply:
No, the git command will still fail since `fetchFromGithub` leaves a dir, not a local repo. I will look into replacing the test target from the Makefile.

Reply:
Actually git is not required at all for this. Will fix.

Reply:
Fixed.